### PR TITLE
Replace early bed ticket with weekend ticket

### PR DESCRIPTION
**Pull request by railsstudent**
1) Comment out Early bird ticket
2) Highlight Full Ticket
3) Place Weekend Ticket between Full Ticket and Student Ticket
4) Change Get Early Bird Ticket at top to Get Full Ticket

### CODE REVIEW

**Comment by tonyhhyip** [comment]
@yookoala Please check for the ga link part

File: assets/pages/index.jinja [hunk 1]
```diff
@@ -28,8 +28,8 @@
                 data-ga-action="click"
                 data-ga-label="ticket"
                 data-ga-value="earlybird ticket"
-                href="{{ data.site.ticketURL | appendURL | withGA('earlybird_ticket', 'frontpage_top') | execAppend }}"
-                target="_blank">Get Early Bird Ticket</a>
+                href="{{ data.site.ticketURL | appendURL | withGA('full_ticket', 'frontpage_top') | execAppend }}"
+                target="_blank">Get Full Ticket</a>
```
Comment (tonyhhyip) [inline assets/pages/index.jinja]: Use `Get Ticket`

**Review by tonyhhyip — DISMISSED** [review]

File: assets/pages/index.jinja (hunk 1 above)
Comment (railsstudent) [reply assets/pages/index.jinja]: Fixed

**Review by yookoala — APPROVED** [review]
Tracking link should be fine.

**Pull request merged**